helm/ako/templates/configmap.yaml

```diff
@@ -27,7 +27,6 @@ data:
   nsxtT1LR: {{ .Values.NetworkSettings.nsxtT1LR | quote }}
   logLevel: {{ .Values.AKOSettings.logLevel | quote }}
   deleteConfig: {{ .Values.AKOSettings.deleteConfig | quote }}
-  advancedL4: {{ .Values.L4Settings.advancedL4 | quote }}
   autoFQDN: {{ .Values.L4Settings.autoFQDN | quote }}
   {{ if .Values.AKOSettings.syncNamespace  }}
```

Comment:
@sudswasavi : Sorry, for asking un-related change to this PR. But variable `syncNamespace` has been not exposed through values.yaml anymore. Can we take out this from AKO repo.
Then, we can ask @RiyaGaniga-avi to take out its presence from AKO operator code for 1.6.1

Reply:
also I think we were supposed to move `autoFQDN` to AKOSettings I believe @sudswasavi